internal/corazarules/rule_match.go

```diff
@@ -125,10 +127,22 @@ func (mr MatchedRule) writeDetails(log *strings.Builder, matchData types.MatchDa
 		mr.ServerIPAddress_, mr.URI_, mr.TransactionID_))
 }
 
-func (mr MatchedRule) matchData(matchData types.MatchData, log *strings.Builder) {
+func (mr MatchedRule) writeExtraRuleDetails(log *strings.Builder, matchData types.MatchData, n int) {
```

Comment:
Can we cover the changes on this file with tests?

Reply:
Added a new `rule_match_test.go` file, Codecov seems to be happy now